### DISCUSSION TITLE
关于依赖问题

### DISCUSSION THREAD

**Discussion by dadanliu** [umijs/umi#11949] [Q&A]
如果我在项目中引用的 package A, A的依赖 package B 由于兼容性问题报错，导致项目直接白屏，这种情况有什么好的解决办法么。现在想到的办法是fork 一份 package A，去更改它的引用指向我更改问题的自定义 package B。不过感觉方法有点笨拙

**Comment by fz6m** [comment]
如果你知道是哪个包有兼容性问题，配置 [`extraBabelIncludes`](https://umijs.org/docs/api/config#extrababelincludes) 让它也被 babel 编译即可，如果需要更高的兼容性，可参考 [非现代浏览器兼容](https://umijs.org/blog/legacy-browser) 。
